From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Sun, 8 Apr 2018 11:15:32 -0400
Subject: goa: adjust description to not mention email or contacts

GNOME doesn't have a core email app and Contacts might not be installed either

Bug: https://bugzilla.gnome.org/show_bug.cgi?id=795077
Forwarded: https://gitlab.gnome.org/GNOME/gnome-initial-setup/merge_requests/13/
---
 gnome-initial-setup/pages/goa/gis-goa-page.ui | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gnome-initial-setup/pages/goa/gis-goa-page.ui b/gnome-initial-setup/pages/goa/gis-goa-page.ui
index c273afc..ae5530d 100644
--- a/gnome-initial-setup/pages/goa/gis-goa-page.ui
+++ b/gnome-initial-setup/pages/goa/gis-goa-page.ui
@@ -10,7 +10,7 @@
               <object class="GisPageHeader" id="header">
                 <property name="margin_top">24</property>
                 <property name="title" translatable="yes">Connect Your Online Accounts</property>
-                <property name="subtitle" translatable="yes">Connect your accounts to easily access your email, online calendar, contacts, documents and photos.</property>
+                <property name="subtitle" translatable="yes">Connect your accounts to easily access your online calendar, documents, photos and more.</property>
                 <property name="icon_name">goa-panel-symbolic</property>
                 <property name="show_icon" bind-source="GisGoaPage" bind-property="small-screen" bind-flags="invert-boolean|sync-create"/>
               </object>
